### PR TITLE
Button

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the Button Component
The U.I sample is attached below

image

### CODE REVIEW

File: Components/buttons/button.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+@import url("styles.css");
+
+
+:root {
+    --default-btn-color: black;
+    --secondary-btn-color: #d8d8da;
+}
+
```
Comment (yashpatel1797) [inline Components/buttons/button.css]: you can declare this in one common CSS and use it in every CSS file.

File: Components/buttons/button.css [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+.btn-outline {
+    background-color: white;
```
Comment (yashpatel1797) [inline Components/buttons/button.css]: if declaring a variable for color then use from there to be consistant.

**Review by yashpatel1797 — COMMENTED** [review]
try to follow the same unit everywhere to be consistent.

**Review by muskan-gupta-0803 — APPROVED** [review]
This looks good. Just a suggestion to remove unnecessary blank lines and commented code in css file.

**Pull request merged**